plugins/modules/pfsense_default_gateway.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Orion Poplawski <orion@nwra.com>
+# Copyright: (c) 2018, Frederic Bor <frederic.bor@wanadoo.fr>
+# Copyright: (c) 2023, Nicolas Zagulajew
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = """
+---
+module: pfsense_default_gateway
+version_added: 0.1.0
+author: Nicolas Zagulajew
+short_description: Manage pfSense default gateway
+description:
+  - Check and update pfSense default gateway
+notes:
+options:
+  name:
+    description: Gateway name
+    required: true
+    type: str
+  ipprotocol:
+    description: Choose the Internet Protocol this gateway uses.
+    required: false
+    choices: [ "inet", "inet6" ]
+    default: inet
+    type: str
+  modify:
+    description: Choose to update default gateway value or not, if not returns current value 
+    default: false
+    type: bool
+"""
+
+EXAMPLES = """
+- name: Sets default gateway to automatic
+  pfsense_gateway:
+    name: Automatic
+    state: present
+    ipprotocol: inet
+
+- name: Remove gateway
```

Comment:
This doesn't seem relevant for `pfsense_default_gateway`.

Reply:
it removes the current default gateway, setting it to "none".. ("-" in the XML)
I fixed and explained.